atomix/cluster/src/main/java/io/atomix/raft/partition/RaftPartitionGroup.java

```diff
@@ -151,62 +144,26 @@ public List<PartitionId> getPartitionIds() {
   }
 
   @Override
-  public PartitionGroupConfig config() {
+  public RaftPartitionGroupConfig config() {
     return config;
   }
 
-  /**
-   * Takes snapshots of all Raft partitions.
-   *
-   * @return a future to be completed once snapshots have been taken
-   */
-  public CompletableFuture<Void> snapshot() {
-    return Futures.allOf(
-            config.getMembers().stream()
-                .map(MemberId::from)
-                .map(id -> communicationService.send(snapshotSubject, null, id, SNAPSHOT_TIMEOUT))
-                .collect(Collectors.toList()))
-        .thenApply(v -> null);
-  }
-
   @Override
   public String toString() {
-    return toStringHelper(this).add("name", name).add("partitions", partitions).toString();
-  }
-
-  /**
-   * Handles a snapshot request from a peer.
-   *
-   * @return a future to be completed once the snapshot is complete
-   */
-  private CompletableFuture<Void> handleSnapshot() {
-    return Futures.allOf(
-            partitions.values().stream()
-                .map(partition -> partition.snapshot())
-                .collect(Collectors.toList()))
-        .thenApply(v -> null);
+    return ToStringBuilder.reflectionToString(this);
```

Comment:
:question: We don't normally use this pattern for toString.

Reply:
I did see we do, actually, and I liked it better than the ToStringHelper here (though it's mostly on the ZPA team's stuff). But I'm happy to use the usual thing :+1: